src/components/Venues/Map.js

```diff
@@ -389,11 +437,16 @@ export default class Map extends React.Component {
             >
               <ButtonContent>
                 <Icon glyph="directionArrow" size={1} color="white" />
-                <p style={{ margin: '0 0 0 0.5rem' }}>
-                  {formatMessage(messages.locateMeButton)}
-                </p>
-              </ButtonContent>
-            </Button>
+                <p
+                  style={{
+                    margin: '0 0 0 0.5rem'
+                  }}
+                >
+                  {' '}
+                  {formatMessage(messages.locateMeButton)}{' '}
```

Comment:
![Screen Shot 2020-09-19 at 7 41 21 PM](https://user-images.githubusercontent.com/40723547/93692401-1a42e080-fab0-11ea-825f-0f06aca9288b.png)
@karimbounekhla Do we need to initialize formatMessage here for it to be recognized?

Reply:
Did you two figure this error out?

Reply:
I updated the PR, it should work now with no error